compose.yml

```
@@ -0,0 +1,45 @@
+services:
+  wazuh-mcp-server:
+    build: .
+    container_name: wazuh-mcp-server
+    restart: unless-stopped
+    init: true
+    
+    # Load configuration from file (created by configure-wazuh.sh)
+    # If file doesn't exist, use environment variables or defaults
+    env_file:
+      - .env.wazuh
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)